app/helpers/input-functions.js

```diff
@@ -0,0 +1,88 @@
+function isChecked (data, option) {
```

Comment:
name of this file is confusing for me , it is better to call it something like answerOptions-functions.js

Reply:
the name ```input-functions``` means functions that work for the inputs we use
why ```answerOptions-functions```?

Reply:
because input-functions is like just functions for input types are here